Bottom: afe1c22145bb8c1c55d373ce5e88ba328a01309d
Top:    f6259598bb05817bffc7257411bf900c1c2758d7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-27 10:46:17 -0500

Add Changepassword.php page to phpldapadmin



---
diff --git a/docker/docker-phpldapadmin/Dockerfile b/docker/docker-phpldapadmin/Dockerfile
index 4544e647d..249051d65 100644
--- a/docker/docker-phpldapadmin/Dockerfile
+++ b/docker/docker-phpldapadmin/Dockerfile
@@ -1,3 +1,3 @@
 FROM osixia/phpldapadmin:latest
 
-COPY changepassword.php /var/www/changepassword.php
+COPY changepassword.php /var/www/phpldapadmin_bootstrap/htdocs/changepassword.php
